titleize/titleize.js

```diff
@@ -1,8 +1,14 @@
+const performTitleCase = sentence => {
+  return sentence.charAt(0).toUpperCase() + sentence.slice(1).toLowerCase();
+}
 
-function titleize(...args) {
-  return args;
+function titleize(sentence) {
+  if (!sentence.split) return sentence;
```

Comment:
Do you think, applying the `.split` makes more sense here? You could directly check `sentence` here.